app/controllers/users_controller.rb

```diff
@@ -5,6 +5,11 @@ def show
     @orders = Order.where(user_id: current_user.id)
   end
 
+  def total_price(order)
+    order.items.inject(0) { |sum, item| sum + item.price }
+  end
+  helper_method :total_price
```

Comment:
I don't think so. You can't run helper methods from the Model, plus this isn't pulling anything from the database.